web/styles/zulip.css

```diff
@@ -1718,12 +1718,6 @@ div.focused_table {
         );
         mask-size: cover;
     }
```

Comment:
Should the rest of this block be moved too?

Reply:
I think it would a good idea to shift the rest of the `.message_content` to `message_row.css` for better understanding of code. I'll make the changes soon.